fixtures/fixtures.go

```diff
@@ -264,20 +264,34 @@ func (g Fixtures) Exclude(tag string) Fixtures {
 	return r
 }
 
-type Suite struct{}
-
-func (s *Suite) SetUpSuite(c *check.C) {
+// InitFixtures set the correct path to be able to access to the fixtures files
+func InitFixtures() {
```

Comment:
Probably `Init` and `Clean` would be better names since we're already in the fixtures packages (e.g. `fixtures.Init()`)